### PR TITLE
Update Pipfile to not include koji_wrapper in dev packages.

### PR DESCRIPTION
**Pull request by jguiditta**
I think it makes more sense for this to be installed via pip -e in the
virtualenv.

Signed-off-by: Jason Guiditta <jason.guiditta@gmail.com>

### CODE REVIEW

**Comment by jguiditta** [comment]
Note, this should also help whatever the issue is with my other patch, as that is purely related to setting up the deps

File: Pipfile.lock [hunk 1]
```diff
@@ -331,6 +321,14 @@
             ],
             "version": "==0.6.1"
         },
+        "more-itertools": {
```
Comment (fuzzball81) [inline Pipfile.lock]: I am assuming this was pulled in as a dep from another package and not something we are using directly.

File: Makefile [hunk 2]
```diff
@@ -91,4 +91,8 @@ install: init ## install the package to the active Python's site-packages
 	pipenv run python setup.py install
 
 dev: init ## set up a development environment
-	pipenv install '-e .' --dev
+	pipenv install --dev
+	pipenv run pip install -e .
+
+reset: clean # clean out your pipenv virtualenv
```
Comment (fuzzball81) [inline Makefile]: I like this!

File: Pipfile.lock (hunk 1 above)
Comment (jguiditta) [reply Pipfile.lock]: Yes, the .lock file is generated by pipenv, nothing manually added.  I did this update with a clean venv that had only the basic pre-installed venv libs in it, like wheel, etc.

**Pull request merged**